client/extensions/wp-job-manager/components/settings/job-submission/index.jsx

```diff
@@ -1,16 +1,17 @@
 /** @format */
-
 /**
  * External dependencies
  */
-
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { change, formValueSelector, FormSection, isDirty, reduxForm } from 'redux-form';
+import reduxFormActions from 'redux-form/es/actions';
```

Comment:
TBH, I don't like the manual change of the import path at all.
This looks to me like on instance of a situation we're going to see more and more: npms publishing their untranspiled code alongside the transpiled code. This is exactly what my #19698 is about (see references to 2ality blog posts there for more background), and I'd really hate to fix (work around, rather) this at individual import paths level.

Even if we don't go with #19698 just yet (tho we should probably discuss that option, too), can we introduce a [Webpack alias](https://github.com/Automattic/wp-calypso/blob/945c1d10b0e06d062fcaddb0f4a937d0ff182603/webpack.config.js#L147-L153) for redux-forms (that points to the `es/` subdir), and add it to [the list of `node_modules` that need to be transpiled](https://github.com/Automattic/wp-calypso/blob/945c1d10b0e06d062fcaddb0f4a937d0ff182603/webpack.config.js#L101) as an alternative approach to this PR?

Reply:
I had a quick look -- `redux-form` definitely [caters](https://github.com/erikras/redux-form/blob/653aca413f536a11330bd1c56d85272cfc441256/package.json#L8) to users of their untranspiled code, tho they use a different nomenclature than I've seen on 2ality. Wonder where that `jsnext:` prefix thing comes from, maybe there's some sort of a spec out there already...

Reply:
(They luckily also use [`module`](https://github.com/erikras/redux-form/blob/653aca413f536a11330bd1c56d85272cfc441256/package.json#L7) for the untranspiled code, so #19698 should actually work OOTB with it.)